### PR TITLE
refactor : about

### PR DESCRIPTION
**Pull request by aswinidev**
Refactored About

- Removed Vuetify Components
- Added TailwindCSS

### CODE REVIEW

**Comment by m-e-l-u-h-a-n** [comment]
@aswinidev can you post a screenshot for this changed card. You can use it in any page for screenshot.

**Comment by aswinidev** [comment]
Screenshot from 2021-05-28 08-38-52

File: components/About.vue [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+  <div class="tw-rounded-md tw-border-2 tw-border-gray-200 tw-shadow-md tw-bg-gray-50">
+    <div class="tw-text-center tw-flex tw-justify-around">
+      <div class="tw-text-center tw-font-medium tw-text-3xl tw-my-2 tw-text-align-bottom">About us</div>
+    </div>
+    <div class="tw-text-center tw-mx-3 tw-px-2">
       <slot name="p1"></slot>
-      <hr class="my-3" />
-    </v-card-text>
-  </v-card>
+      <hr class="tw-my-3" />
```
Comment (m-e-l-u-h-a-n) [inline components/About.vue:8]: I think this can be removed. As card already has its own shadow.

**Review by m-e-l-u-h-a-n — CHANGES_REQUESTED** [review]
I went through the changes and posted a minor comment. There are also linting error. that need to be addressed.
Rest looks fine 🎉.

**Comment by aswinidev** [comment]
I made the requested changes and also fixed the linting error

**Review by m-e-l-u-h-a-n — APPROVED** [review]

**Comment by m-e-l-u-h-a-n** [comment]
LGTM, Merged🎊!
Thanks for working on it.

**Pull request merged**